frontend/src/components/Slider/NamedSlider.tsx

```diff
@@ -0,0 +1,76 @@
+import { ReactNode, useState, MouseEventHandler, useRef, KeyboardEventHandler } from 'react';
```

Comment:
Please add your components to storybook.

Reply:
I created a story for the slider. I think we should create stories for all the general components (components in src/components) but we should create a separate issue for that.

Reply:
Issues are very patient, it is much better to always introduce documentation with PRs.

Reply:
The components I meant have nothing to do with this PR. I documented the NamedSlider which I introduced developing this feature. I just observed that none of the other existing components are documented yet except the Buttons if I recall correctly. This should be done in a fresh PR as it gets very confusing otherwise.
The questions is, if the bigger components (the whole toolbar for example) should also be documented in storybook? This of course will be done in this PR if it is wanted.

Reply:
Thanks for creating #333.